.github/workflows/build_and_test.yml

```diff
@@ -299,6 +299,7 @@ jobs:
             host-arch: x86_64
             arch: arm64
             config-zstd: OFF
+            additional-flags: -DCMAKE_CROSSCOMPILING=1 -DRUN_HAVE_STD_REGEX=0
```

Comment:
Otherwise cross-compile fails because of Google Benchmark. It can't detect regex implementation somehow. Check the build log https://github.com/seladb/PcapPlusPlus/actions/runs/11327526535/job/31498794905. The workaround mentioned here: https://github.com/google/benchmark/issues/351

Reply:
Is there a way to avoid compiling Google Benchmark if regex implementation isn't found or if `CMAKE_CROSSCOMPILING` is turned on? 🤔